source/getting-started/building-for-production.md

```diff
@@ -0,0 +1,341 @@
+Ember can automatically build assets for production environments. In these cases,
+assets will be minified, versioned and the production versions of libraries will be
+used.
+
+### Raw Assets
+
+* `public/assets` vs `app/styles`
+
+To add images, fonts, or other assets, place them in the `public/assets` directory. For
+example, if you place `logo.png` in `public/assets/images`, you can reference it in
+templates with `assets/images/logo.png` or in stylesheets with
+`url('/assets/images/logo.png')`.
+
+### Minifying
+
+The compiled css-files are minified by `broccoli-clean-css` or `broccoli-csso`,
+if it is installed locally. You can pass minifer-specific options to them using
+the `minifyCSS:options` object in your Brocfile. Minification is enabled by
+default in the production-env and can be disabled using the `minifyCSS:enabled`
+switch.
+
+Similarly, the js-files are minified with `broccoli-uglify-js` in the
+production-env by default. You can pass custom options to the minifier via the
+`minifyJS:options` object in your Brocfile. To enable or disable JS minification
+you may supply a boolean value for `minifyJS:enabled`.
+
+For example, to disable minifying of CSS and JS, add in `Brocfile.js`:
+```Brocfile.js
+minifyCSS: {
+  enabled: false
+},
+minifyJS: {
+  enabled: false
+}
+```
+
+### Source Maps
+
+Ember CLI supports producing source maps from your CSS and JS files.
+Source maps are configured by the `sourcemaps` option and
+are disabled in production by default.
+
+In dev, the default setting is equal to:
+
+```Brocfile.js
+sourcemaps: {
+  enabled: true,
+  extensions: ['js']
+}
+```
+
+Pass `{enabled: true}` to your EmberApp constructor to enable source maps for javascript.
+Use the `extensions` option to configure CSS.
+
+### Stylesheets
```

Comment:
I think lines 55-69 should be moved to the Managing Dependencies page (with slight modifications to make things flow properly).